charts/argocd-apps/values-test.yaml

```diff
@@ -269,3 +269,18 @@ applications:
         value: https://www.gov.uk
       - name: GOVUK_APP_NAME
         value: info-frontend
+- name: collections
+  helmValues:
+    appImage:
+      repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/collections
+      tag: latest  # To be edited by automation (not yet implemented).
+    replicaCount: 1
+    extraEnv:
+      - name: SECRET_KEY_BASE
+        value: secret  # TODO: set SECRET_KEY_BASE and make it a secret
+      - name: GOVUK_APP_DOMAIN
+        value: www.gov.uk
+      - name: GOVUK_WEBSITE_ROOT
+        value: https://www.gov.uk
```

Comment:
These can be removed as they are covered by govuk-apps-conf default Values config map